Makefile

```diff
@@ -50,6 +50,11 @@ pylint:  ## lint
 coverage:  ## Run coverage, without the built-in virtualenv
 	@. ve/bin/activate; coverage run --source figures -m py.test; coverage report -m
 
+ginkgo.pytest:  ## Run Pytest for the Ginkgo environment
+	OPENEDX_RELEASE=GINKGO pytest -c pytest-ginkgo.ini
+
+ginkgo.tox:  ## Run tox just for the Ginkgo environment
+	tox -e py27-ginkgo
```

Comment:
I can't run any of the `make` commands, failing on 

```
Makefile:5: .env*: No such file or directory
```
Are those dotenv files generated or specified somehow else that I'm not seeing, or are those particular to your dev environment @johnbaldwin ?  Is there any requied configuration that should be stored in them?  Shouldn't this not fail if missing?

Reply:
Just need a look at this @johnbaldwin and I'll approve :D

Reply:
@bryanlandia I've been building the makefile as a helper, so I think you are the first other person to run it. There should be a default .env file, I'll add as a different PR. To note, TravisCI does not use the makefile.

 Please ignore this for this PR and just run this:

`tox -e py27-ginkg`

Reply:
But if you are curious, the `.env` file looks something like this:

```
# This file provides default environemnt settings
# override by creating a '.env.<some-label>' file for the specific settings
# you wish to override

# Set the edx-platform named release
EDX_PLATFORM_RELEASE=hawthorn
```

Reply:
@johnbaldwin OK, sounds fine.  I did run that `tox` command.